tests/fields/fields.py

```diff
@@ -949,6 +956,17 @@ class BlogPost(Document):
         post.tags = ('fun', 'leisure')
         post.validate()
 
+        post.access_list = 'a,b'
+        self.assertRaises(ValidationError, post.validate)
+
+        post.access_list = ['c', 'd']
+        self.assertRaises(ValidationError, post.validate)
+
+        post.access_list = ['a', 'b']
+        post.validate()
+
+        self.assertEqual(post.get_access_list_display(), u'Administrator,Manager')
```

Comment:
Shouldn't need `u` and, if you follow the advice from my comment above, change this to `'Administration, Manager'`.